getDataTool function failing - 404 error

Azure Cosmos DB Migration Tool Task fails with "Error: Parameter 'toolPath' cannot be null or empty."

running the release with System.Debug = true shows a 404 error when trying download the migration tool zip file:
2022-10-16T19:40:53.8790422Z ##[debug]Failed to download "5c797be1-b723-4cfc-889f-1267e3ae51c3" from "https://download.microsoft.com/download/E/1/4/E143A339-41AE-4E0E-9CC8-911C0B663478/dt-1.7.zip". Code(404) Message(Not Found)
2022-10-16T19:40:53.8793291Z ERR:Unexpected HTTP response: 404
2022-10-16T19:40:53.8793872Z ##[debug]task result: Failed

Having the same issue:

==============================================================================
Task         : Cosmos DB Data Migration Tool
Description  : With the Azure Cosmos DB Data Migration tool, you can easily migrate data to Azure Cosmos DB
Version      : 1.0.13
Author       : Winvision BV
Help         : [Support](https://github.com/Winvision/vsts-tasks)
==============================================================================
Downloading: https://download.microsoft.com/download/E/1/4/E143A339-41AE-4E0E-9CC8-911C0B663478/dt-1.7.zip
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6768) Warning: Use Cipheriv for counter mode of aes-256-ctr

**ERR:Unexpected HTTP response: 404
##[error]Error: Parameter 'toolPath' cannot be null or empty.**

Any one have solution for that facing the same issue.
##[debug]Failed to download "0f6a0108-de44-4db0-97fc-b277bcccc154" from "https://download.microsoft.com/download/E/1/4/E143A339-41AE-4E0E-9CC8-911C0B663478/dt-1.7.zip". Code(404) Message(Not Found)

ERR:Unexpected HTTP response: 404
##[error]Error: Parameter 'toolPath' cannot be null or empty.

--- a/cosmosdb-tasks/Tasks/DataMigrationTool/datatoolDownloader.ts
+++ b/cosmosdb-tasks/Tasks/DataMigrationTool/datatoolDownloader.ts
@@ -10,9 +10,7 @@ export async function getDataTool(versionSpec: string, addToolToPath?: boolean):
     toolPath = toolLib.findLocalTool('cosmosdt', versionSpec);
 
     if (!toolPath) {
-        const version = semver.clean(versionSpec);
-        const v = `${semver.major(version)}.${semver.minor(version)}`;
-        const url = `https://download.microsoft.com/download/E/1/4/E143A339-41AE-4E0E-9CC8-911C0B663478/dt-${v}.zip`;
+        const url = `https://github.com/Azure/azure-documentdb-datamigrationtool/releases/download/1.8.3/azure-documentdb-datamigrationtool-1.8.3.zip`;
 
         taskLib.debug('Downloading version: ' + url);
         const downloadPath: string = await toolLib.downloadTool(url);

